packages/expect/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "jest-matchers",
+  "name": "expect",
```

Comment:
Any concrete features you can think of?

Mocking is provided by `jest-mock`, which will need docs of course, but I think there is feature parity.